src/treemodel.cpp

```diff
@@ -91,6 +91,26 @@ QVariant TreeModel::data(const QModelIndex &index, int role) const
             return item->Data(col);
             break;
         }
+        case Qt::ToolTipRole:
+        {
+            if (col == COL::Value)
+            {
+                const int ToolTipLimit = 500;
```

Comment:
These consts are not used anywhere else and likely won't be. They are used merely for readability and quick tuning, so I think keeping them locally is easier for those purposes.